src/components/ProjectPage/ProjectPage.js

```diff
@@ -106,7 +106,7 @@ class ProjectPage extends PureComponent<Props> {
           <DevelopmentServerPane leftSideWidth={300} />
 
           <Spacer size={30} />
-          <TaskRunnerPane leftSideWidth={200} />
+          <TaskRunnerPane leftSideWidth={200} projectId={project.id} />
```

Comment:
Right. So I did this while following the [Reselect docs](https://github.com/reduxjs/reselect#accessing-react-props-in-selectors), figuring maybe it mattered if it was done as the raw `ownProps` object instead of a newly-created object. I just did some testing, though, and it doesn't seem to really matter; I can't tell any difference in terms of the selector recomputing.

So yeah I'll revert this change, I agree it's nicer to use `mapStateToProps` to get the current project